manifests/kubeapps.jsonnet

```diff
@@ -18,7 +18,11 @@ local labels = {
 // the manifest key
 local labelify(src) = if std.objectHas(src, "metadata") then src + labels else src;
 local labelifyEach(src) = {
-  [k]: labelify(src[k]) for k in std.objectFields(src)
+  [k]: if std.isArray(src[k]) then
```

Comment:
Does this also fix https://github.com/kubeapps/kubeapps/issues/151?

Reply:
If that issue refers to the `kubeless-controller` it was already setting the label `created-by: kubeapps` (and it keeps being that way). In case it was referring to the function pods that kubeless creates they won't include that tag.

Reply:
I thought @sebgoa was referring to the `kubeless-controller`, but now that you say that maybe he was referring to the function pods, and of course they won't include that tag.